index.js

```diff
@@ -29,6 +29,19 @@ program
             .catch((err) => console.error(err.message))
     });
 
+program
+  .command('dapp:build <signing-key-file>')
+  .option('-d, --dev', 'Dev mode build')
+  .description('Build a DApp to an installable bundle')
```

Comment:
thats not correct. This branch is about streaming it. Not about building a stand alone version.